lib/torrent.js

```diff
@@ -750,7 +751,7 @@ class Torrent extends EventEmitter {
       }
     }
 
-    const wasAdded = !!this._addPeer(peer)
+    const wasAdded = !!this._addPeer(peer, this.client.utp ? 'utp' : 'tcp')
```

Comment:
just a note for other reviewers: if the utp connection fails to connect, then it is replaced with a tcp connection to the same ip:port

maybe a comment here explaining that would be good

Reply:
Right! I've added a comment 👍